Poisson_surface_reconstruction_3/include/CGAL/poisson_refine_triangulation.h

```diff
@@ -180,17 +180,17 @@ class Poisson_mesher_level :
 /// bad means badly shaped or too big).
 /// @return the number of vertices inserted.
 ///
-/// @commentheading Preconditions:
+/// *Preconditions:*
 /// - Tr must use a geometric traits with robust circumcenter computation.
 /// - convergence is guaranteed if radius_edge_ratio_bound >= 1.0.
 ///
-/// @commentheading Template Parameters:
-/// @param Tr 3D Delaunay triangulation.
-/// @param Surface Sphere_3 or Iso_cuboid_3.
-/// @param Sizing_field A sizing field functor type
-/// @param Second_sizing_field A sizing field functor type
+/// *Template Parameters*
```

Comment:
Is this line not a bit overdone as doxygen will add a header **Template Parameters**